selenium/src/test/java/au/org/emii/portal/tests/landing/AcknowledgementTest.java

```diff
@@ -0,0 +1,17 @@
+package au.org.emii.portal.tests.landing;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+public class AcknowledgementTest extends BaseTest {
+
+    @Test
+    public void linkTest() {
+        webElementUtil.clickLinkContainingText("Acknowledgement");
+        webElementUtil.switchToNewTab();
+        webElementUtil.verifyPageTitle("Data Use Acknowledgement | Portal User Guide");
```

Comment:
Can groovy environment variables be used in the comparison?
Is it worth (or possible) importing the en.js file so it could compare strings that are used from it?

I'm thinking these tests are all so brittle